HW7/YuraVladyka/hw3.py

```diff
@@ -0,0 +1,16 @@
+def counting_letters(string):
```

Comment:
good job
see here a more optimized version of the code
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result